google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/ProxyService.java

```diff
@@ -57,6 +57,14 @@ protected final <T extends ApiService> void addServices(Collection<T> services)
     checkState(state() == State.NEW);
     for (ApiService service : services) {
       checkArgument(service.state() == State.NEW, "All services must not be started.");
+      service.addListener(
+          new Listener() {
```

Comment:
Nit: use the same listener instance for all the dependencies?

Reply:
Done.